components/automate-ui/src/app/pages/integrations/list/integrations-list.component.spec.ts

```diff
@@ -1,16 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { RouterTestingModule } from '@angular/router/testing';
-import { StoreModule } from '@ngrx/store';
+import { StoreModule, Store } from '@ngrx/store';
 
-import { runtimeChecks, ngrxReducers } from 'app/ngrx.reducers';
+import { runtimeChecks, ngrxReducers, NgrxStateAtom } from 'app/ngrx.reducers';
 import { ChefPipesModule } from '../../../pipes/chef-pipes.module';
```

Comment:
Please remove the parent (..) indicators when you might encounter them.
```suggestion
import { ChefPipesModule } from 'app/pipes/chef-pipes.module';
```